<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroPage</class>
 <widget class="QWizardPage" name="introPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Paso 1: Configuración Inicial</string>
  </property>
  <property name="subTitle">
   <string>Defina el nombre del caso y seleccione una plantilla base para la simulación.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Bienvenido al asistente. Asigne un nombre único a su caso y elija una de las plantillas disponibles para comenzar. Los parámetros específicos podrán ser editados más adelante.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="caseNameLabel">
       <property name="text">
        <string>Nombre del Caso:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="caseNameLineEdit">
       <property name="placeholderText">
        <string>Ej: mi_simulacion_canal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="creationModeGroupBox">
     <property name="title">
      <string>Método de Creación</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="templateRadioButton">
        <property name="text">
         <string>Usar una plantilla</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="customFilesRadioButton">
        <property name="text">
         <string>Seleccionar archivos individualmente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="creationModeStackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="templateContainer">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="templateLabel">
            <property name="text">
             <string>Plantilla:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="templateComboBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="customFileContainer">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="availableFilesLayout">
            <item>
             <widget class="QLabel" name="availableFilesLabel">
              <property name="text">
               <string>Archivos Disponibles:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="availableFilesListWidget"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="addRemoveLayout">
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeButton">
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="selectedFilesLayout">
            <item>
             <widget class="QLabel" name="selectedFilesLabel">
              <property name="text">
               <string>Archivos Seleccionados:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="selectedFilesListWidget"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
